SCons/Tool/__init__.py

```diff
@@ -130,62 +130,62 @@ def _tool_module(self):
         """
         oldpythonpath = sys.path
         sys.path = self.toolpath + sys.path
-        # sys.stderr.write("Tool:%s\nPATH:%s\n" % (self.name,sys.path))
-        # sys.stderr.write("toolpath:%s\n" % self.toolpath)
-        # sys.stderr.write("SCONS.TOOL path:%s\n" % sys.modules['SCons.Tool'].__path__)
+        # These could be enabled under "if debug:"
+        # sys.stderr.write(f"Tool: {self.name}\n")
+        # sys.stderr.write(f"PATH: {sys.path}\n")
+        # sys.stderr.write(f"toolpath: {self.toolpath}\n")
+        # sys.stderr.write(f"SCONS.TOOL path: {sys.modules['SCons.Tool'].__path__}\n")
         debug = False
         spec = None
         found_name = self.name
         add_to_scons_tools_namespace = False
 
-
         # Search for the tool module, but don't import it, yet.
         #
         # First look in the toolpath: these take priority.
         # TODO: any reason to not just use find_spec here?
         for path in self.toolpath:
             sepname = self.name.replace('.', os.path.sep)
-            file_path = os.path.join(path, "%s.py" % sepname)
+            file_path = os.path.join(path, sepname + ".py")
             file_package = os.path.join(path, sepname)
 
-            if debug: sys.stderr.write("Trying:%s %s\n" % (file_path, file_package))
+            if debug: sys.stderr.write(f"Trying: {file_path} {file_package}\n")
```

Comment:
> __[Flake8]__ multiple statements on one line (colon) ([view](https://sider.review/gh/repos/104670160/pulls/4201#issue-7388438))
>
> <details>
>
> | Rule |
> | ---- |
> | `E701` |
>
> You can [close this issue](https://sider.review/gh/repos/104670160/pulls/4201#issue-7388438) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

I know if we Black the codebase these debug things will be split, but I'm explicitly trying to keep them on one line at least for now (the Trace calls elsewhere also are written this way).